src/Core/ClassInfo.php

```diff
@@ -4,12 +4,14 @@
 
 use Exception;
 use ReflectionClass;
-use SilverStripe\CMS\Model\SiteTree;
-use SilverStripe\Control\Director;
-use SilverStripe\Core\Manifest\ClassLoader;
-use SilverStripe\ORM\DataObject;
 use SilverStripe\ORM\DB;
+use SilverStripe\ORM\DataObject;
+use SilverStripe\Control\Director;
+use Psr\SimpleCache\CacheInterface;
 use SilverStripe\View\ViewableData;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Core\Manifest\ClassLoader;
```

Comment:
Please avoid reordering `use` statements like this. We don't have a set standard for the order, so this is just adding extra work for me to review that all that's changed here is the ordering (and in this case the addition of `Psr\SimpleCache\CacheInterface`).

This isn't the first time I've asked this.

If your IDE is automatically making these changes, you may need to change your IDE settings (perhaps have a separate profile if your IDE has that functionality) for when you're making contributions.